styles/style.css

```diff
@@ -92,11 +96,37 @@ p {
   border-left-color: var(--orange-dark);
 }
 
-
 /* Buttons */
 
-  /* INSERT BUTTON STYLES HERE */
+/* INSERT BUTTON STYLES HERE */
+
+button {
+  background-color: #e95b1e;
+  color: white;
+  border-radius: 4px;
+  padding: 6px 6px;
+  border: 0px;
+}
 
+button:hover {
+  background-color: rgb(55, 178, 235);
+}
+.orange {
+  background-color: #e95b1e;
+  color: white;
+  margin-left: 25px;
+  border-radius: 4px;
+  padding: 6px 6px;
+  border: 0px;
+}
+
+.white {
+  color: #c05326;
+  background-color: white;
+  border-radius: 4px;
+  padding: 6px 6px;
+  border: 0px;
+}
```

Comment:
You have successfully applied the desired styles to your buttons. However, I wanted to bring up an important concept in programming called DRY code, which stands for "Don't Repeat Yourself."
In your code, I see that you have repeated the same set of styles for different button classes, such as "orange" and "white."
Next time, you can consider creating a single CSS class that contains the common styles like;
button { //common styles
  border-radius: 4px;
  padding: 6px 6px;
  border: 0px;
}
.orange {
  background-color: #e95b1e;
  color: white;
  margin-left: 25px;
}
//same idea for the white one